rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/dto/RestMapPage.java

```diff
@@ -0,0 +1,47 @@
+package org.ow2.proactive_grid_cloud_portal.scheduler.dto;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00120)